spec/requests/manager_set_owner_spec.rb

```diff
@@ -1,24 +1,81 @@
 require 'rails_helper'
 
-describe 'Manager sets owner in register' do
-  it 'POST /residents/:resident_id/owners' do
-    resident = create(:resident, full_name: 'Adroaldo Silva')
+describe 'Manager se owner' do
+  context 'DELETE /residents/:resident_id/owners/:id' do
+    it 'and hes not authenticated' do
```

Comment:
Não que precise mudar, mas vocês podem colocar "and he's not authenticated" que o rubocop entende ser necessárias as aspas duplas.

O que vale para as demais ocorrências